snuba/cli/replacer.py

```diff
@@ -84,8 +84,10 @@ def replacer(
         TransportError,
         build_kafka_consumer_configuration,
     )
-    from snuba.utils.streams.batching import BatchProcessingStrategyFactory
-    from snuba.utils.streams.processing import StreamProcessor
+    from snuba.utils.streams.processing.processor import StreamProcessor
```

Comment:
Any reason not to expose StreamProcessor directly in `snuba.utils.stresms.processing` ? It seems to me that additional `processor` module is unlikely to contain much else that should be exported.